core/client/src/main/java/alluxio/client/file/options/CreateFileOptions.java

```diff
@@ -142,6 +153,16 @@ public CreateFileOptions setTtl(long ttl) {
   }
 
   /**
+   * @param ttlExpiryAction the {@link TtlExpiryAction};
```

Comment:
style `ttlExpiryAction the {@link TtlExpiryAction} to use`

(I would not enumerate the types of actions here as that information will become stale over time)